community/2025/survey.yaml

```
@@ -589,9 +600,7 @@ questions:
       - Donor perks (e.g. recognition on website, Discourse badges, swag, exclusive access)
       - The ability to deduct donations from tax
       - More options for ear-marking donations
-      - Official support or services
-      - A maintained Long-Term-Support release
-  - prompt: Which areas should the NixOS Foundation prioritise funds for?
+  - prompt: Which areas should the NixOS Foundation prioritise excess funds for?
```

I think when we reviewed this initially we wanted to add "scarcity" to improve the quality of the answers

```suggestion
  - prompt: Which five areas should the NixOS Foundation prioritise excess funds for?
```

Open to other ideas or leaving as is.

---

The foundation has asked that we leave the wording as it was originally submitted. @infinisil it's your call

---

How about we make this a ranking instead. Don't think we need to limit the number then, and we get more usable data.

---

Sounds good. I'll update the PR and get this into the limesurvey today.